src/pomodoro/containers/PomodoroTitle.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import format from 'date-fns/format'
+import { Helmet } from 'react-helmet'
+import { connect } from 'react-redux'
+import * as fromPomodoro from 'pomodoro/store'
+
+const PomodoroTitle = ({ lapse, title }) => (
+  <Helmet>
+    <title>🕑 {format(lapse, 'mm:ss')} 🗒️ {title}</title>
```

Comment:
Now that we have the `format` function being used in two different places, what do you about creating a new component called `<Format />`, that receives a timestamp as children, and just return the result of the `format` function?